publish.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
This won't work for a standard Windows world -- maybe that's ok but there are some cross platform options: http://www.pyinvoke.org/

Not blocking, just a thought